.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+<!--
+Thanks for opening a PR! Your contribution is much appreciated.
+In order to make sure your PR is handled as smoothly as possible we request that you follow the checklist sections below.
+Choose the right checklist for the change that you're making:
+-->
+
+### Checklist
+Before you submit a pull request, please check:
+
+- [ ] Related issue is linked using `fixes #number`
+- [ ] This changes were tested.
+- [ ] This PR DOES NOT contain breaking changes.
+- [ ] This PR DOES contain breaking changes.
```

Comment:
```suggestion
- [ ] This PR contains breaking changes.
```
Since these are mutually exclusive events, we can simplify it by just adding one check that tells if changes are breaking. If not, we can assume they are not breaking. Unless including both was intentional to make sure people are clear when they are submitting breaking changes?

Reply:
Sure!